lib/fizzy/execute.cpp

```diff
@@ -18,6 +19,8 @@ struct LabelContext
     size_t stack_height = 0;             ///< The stack height at the label instruction.
 };
 
+using LabelsStack = std::stack<LabelContext, std::vector<LabelContext>>;
```

Comment:
similar to "OperandStack" and 
```suggestion
using LabelStack = std::stack<LabelContext, std::vector<LabelContext>>;
```